### ISSUE TITLE
[CAL-2450] Request Payment After Approval

### ISSUE THREAD

**Issue by mywebmasteruk** [calcom/cal.com#11264]
**What we have now**
When integrating payment request, we currently have two options; On booking or Charge a fee on no-show.

**What we need**
It would be amazing to have, a new option, to request payment, AFTER approval (when Requires Confirmation is active)

**Example use case**

Customer would like to book a Consultation session, which costs 200 USD.
I receive a booking request, which I must review before accepting.
I approve the request.
Customer receives notification that the request has been approved and in the same email, a link to make payment.
Customer makes payment, and the booking is confirmed.

<sub>[CAL-2450](https://linear.app/calcom/issue/CAL-2450/request-payment-after-approval)</sub>

**Comment by PeerRich** [comment]
i like this. we'll look into this!

**Comment by Kfelts** [comment]
Just a suggestion - I think this could be enhanced a bit. 
One way is when a person book the consult, it creates a payment intent with Stripe and then places the payment on hold. Once the appointment is accepted, it should release the hold on the payment.

This avoids unnecessary refunds and fees from refunding or payment processing. It also is helpful for those of us that make confirming an appointment contingent on payment.

**Comment by giovannicocco** [comment]
> Just a suggestion - I think this could be enhanced a bit. One way is when a person book the consult, it creates a payment intent with Stripe and then places the payment on hold. Once the appointment is accepted, it should release the hold on the payment.
> 
> This avoids unnecessary refunds and fees from refunding or payment processing. It also is helpful for those of us that make confirming an appointment contingent on payment.

Great suggestion and reduce payment friction
